### PR TITLE
[INT-139] Project rename

### PR DESCRIPTION
**Pull request by Martoon-00**
## Description

It has been decided to rename the project to `xrefcheck`.

## :white_check_mark: Checklist for your Pull Request

Ideally a PR has all of the checkmarks set.

If something in this list is irrelevant to your PR, you should still set this
checkmark indicating that you are sure it is dealt with (be that by irrelevance).

#### Related changes (conditional)

- Tests
  - [x] If I added new functionality, I added tests covering it.
  - [x] If I fixed a bug, I added a regression test to prevent the bug from
        silently reappearing again.

- Documentation
  - [x] I checked whether I should update the docs and did so if necessary:
    - [README](/README.md)
    - Haddock

- Public contracts
  - [x] Any modifications of public contracts comply with the [Evolution
  of Public Contracts](https://www.notion.so/serokell/Evolution-of-Public-Contracts-2a3bf7971abe4806a24f63c84e7076c5) policy.
  - [x] provided a migration guide for breaking changes if possible

#### Stylistic guide (mandatory)

- [x] My commits comply with [the policy used in Serokell](https://www.notion.so/serokell/Where-and-how-to-commit-your-work-58f8973a4b3142c8abbd2e6fd5b3a08e).
- [x] My code complies with the [style guide](docs/code-style.md).

### CODE REVIEW

**Comment by gromakovsky** [comment]
> we will perform an actual project movement later.

Maybe let's do it straight away? Nothing will be broken for anyone anyway because old links will still be working.

File: src/Crv/CLI.hs [hunk 1]
```diff
@@ -109,7 +109,7 @@ totalParser = asum
   ]
 
 versionOption :: Parser (a -> a)
-versionOption = infoOption ("crossref-verify-" <> (showVersion version)) $
+versionOption = infoOption ("xrefcheck-" <> (showVersion version)) $
```
Comment (gromakovsky) [inline src/Crv/CLI.hs]: Minor: you can remove redundant brackets while you are at it.

**Comment by gromakovsky** [comment]
Module names are not that big, so I think starting them with `Xrefcheck` (or `XRefCheck`) would be fine with regards to length (each module name would still be not big), but at the same time much more descriptive than `Xrc`.

**Comment by Martoon-00** [comment]
> > we will perform an actual project movement later.
> 
> Maybe let's do it straight away? Nothing will be broken for anyone anyway because old links will still be working.

Ah okay, I misunderstood what was told in the ticket description.

**Comment by Martoon-00** [comment]
I forced-push all requested changes.

**Comment by gromakovsky** [comment]
Well, the second item there clearly says "Rename the repo" :)

**Comment by gromakovsky** [comment]
> Note that references to the github repository remain untouched, we will perform an actual project movement later.

Should be removed I suppose.

**Comment by gromakovsky** [comment]
Meh, in the first commit you've added `src/Xrefcheck/CLI.hs`

File: src/Xrefcheck/CLI.hs [hunk 2]
```diff
@@ -109,7 +109,7 @@ totalParser = asum
   ]
 
 versionOption :: Parser (a -> a)
-versionOption = infoOption ("xrefcheck-" <> showVersion version) $
+versionOption = infoOption ("xrefcheck-" <> (showVersion version)) $
```
Comment (gromakovsky) [inline src/Xrefcheck/CLI.hs]: Some mess happened: you removed these brackets and then added them back.

Comment (Martoon-00) [reply src/Xrefcheck/CLI.hs]: Wtf, git failed into move + fix.

**Comment by Martoon-00** [comment]
Okay, should be fine now, nice catch.

**Pull request merged**